### ISSUE TITLE
Wish: Add ability to import GPX file via URL

### ISSUE THREAD

**Issue by pwagland** [merlos/iOS-Open-GPX-Tracker#65]
If I am on my desktop, then I can easily download a GPX file, and get in some fashion or another to be able to be read by GPX reader, however, when on vacation, I sometimes only have my iOS devices, and downloading a URL is not always easy or even possible for me. It would be great if I could just paste a URL, and get GPX Reader to open a GPX file from that location.

**Comment by merlos** [comment]
Hi @pwagland. 

Thanks for your message. 

In last version (1.5) that feature was partially added. If you open a GPX file from Safari, it will prompt you to open Open GPX Tracker. It will save to the gpx file to your GPX folder, but you have to open it manually (touch on the top left folder icon, and the first file in the list is the one you imported).

For example browse to https://www.mapbox.com/help/define-gpx/ scroll down and click on the Download a sample GPX file button.
![img_2544](https://user-images.githubusercontent.com/404446/50677846-10bad180-0fca-11e9-8097-62ac459f2af3.PNG)
![img_2545](https://user-images.githubusercontent.com/404446/50677844-0d274a80-0fca-11e9-9fb3-445c86dc30c2.PNG)


So, import is already done, buy confusing because the app does not provide any feedback to the user. Ideally, the file should be automatically opened after import.

Thanks for your comment. 
Best, Juan.

**Comment by pwagland** [comment]
Right, for me it just shows the text of that sample route, it doesn’t show the icon of the application. I have seen elsewhere that this isn’t 100% reliable, so maybe it’s jyst the version of software that I have, I’m running iOS 12.1.2, and open GPX 1.5.0.

**Comment by merlos** [comment]
Thanks @pwagland 

Have you tried to delete the app and install it again? 

Before deleting the app, you can back your GPX files up with iTunes (https://github.com/merlos/iOS-Open-GPX-Tracker/issues/37).

**Comment by vincentneo** [comment]
I remembered that the website itself will have the control on whether to download or open in a browser. Some gets displayed as a website, some gets displayed as a document(if it downloads on desktop). Reinstalls won't work.

For example, the sample from Garmin http://developer.garmin.com/garmin-connect-api/sample-data/ does not get downloaded even from a desktop, but another sample GPX file from MapBox https://www.mapbox.com/help/glossary/gpx/ downloads fine from both devices.

**Comment by merlos** [comment]
Thanks @pwagland  and @vincentneo 

It´s weird. In my case, in both links shared by @vincentneo, Safari prompts me to open Open GPX tracker app.

I used Safari in an iPhone with iOS 12.1.2 (16C101) and in an iPad with 12.0.1 (16A404)...

**Comment by wallstudios** [comment]
Was have a similar issue where I modified a gpx file to merge a couple of trails. I then exported using Basecamp. The file would not load or show in the list. Turned out it was an issue with the case of the extension. 

Basecamp exported it and saved as .GPX (uppercase) and when I changed to lowercase (.gpx) and then airdropped again it would load.
